### PR TITLE
Prepare changelog for 1.3.2 release

### PR DESCRIPTION
**Pull request by jack-berg**

### CODE REVIEW

File: CHANGELOG.md
Comment (jack-berg) [inline CHANGELOG.md:7]:
```suggestion
## 1.3.2 - 2024-05-31
```

**Review by Kielek — APPROVED** [review]

**Review by lmolkova — APPROVED** [review]

File: CHANGELOG.md
Comment (arminru) [inline CHANGELOG.md:7]:
```suggestion
## 1.3.2 - 2024-06-28
```

**Review by arminru — APPROVED** [review]

File: CHANGELOG.md
Comment (arminru) [inline CHANGELOG.md:5]:
```suggestion
The full list of changes can be found in the compare view for the respective release at <https://github.com/open-telemetry/opentelemetry-proto/releases>.
```

Otherwise markdown-link-check will block the PR since the tag won't exist yet 🙂

**Comment by arminru** [comment]
@open-telemetry/profiling-maintainers you can find the new release here: https://github.com/open-telemetry/opentelemetry-proto/releases/tag/v1.3.2

**Pull request merged**
